modules/site-health/audit-database/class-perflabdbmetrics.php

```diff
@@ -0,0 +1,433 @@
+<?php
+/**
+ * Performance Lab database metrics computation module
+ *
+ * @package performance-lab
+ * @group audit-database
+ *
+ * @since 1.4.0
+ */
+
+/**
+ * Performance Lab database metrics class,
+ *
+ * @package performance-lab
+ * @group audit-database
+ *
+ * @since 1.4.0
+ */
+class PerflabDbMetrics {
+	/** First eligible db version, the advent of utfmb4 in WordPress databases.
```

Comment:
Please adjust these docblocks throughout the PR. The description must always only start on the next line.
```suggestion
	/**
	 * First eligible db version, the advent of utfmb4 in WordPress databases.
```